main/src/org/jetbrains/bsp/bazel/BazelBspServer.java

```diff
@@ -859,7 +741,7 @@ private String convertOutputToPath(String output, String prefix) {
 
             int returnCode = parseProcess(process);
             return Either.forRight(new TestResult(BepServer.convertExitCode(returnCode)));
-          } catch (IOException | InterruptedException | ExecutionException e) {
+          } catch (IOException | InterruptedException e) {
```

Comment:
why was ExecutionException removed?

Reply:
Because Semaphore doesn't throw it (`CompletableFuture` does).